publish-release.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+if [[ -z "$GITHUB_TOKEN" ]]; then
+  echo "GITHUB_TOKEN not found in environment"
+  exit 1
+fi
+
+TAG=$1
+if [[ -z "$TAG" ]]; then
+  echo "Expected release tag to be passed as the first argument"
+  exit 1
+fi
+
+if [[ ! "$TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+  echo "Invalid release tag: $TAG"
+  exit 1
+fi
+
+echo "Generating grpcwebproxy binaries"
+cd go
+rm -rf dist
+mkdir -p dist
+
+GOOS=linux GOARCH=amd64 go build -o "dist/grpcwebproxy-$TAG-linux-amd64" ./grpcwebproxy
```

Comment:
Sorry I didn't mention this before, but I think this is usually called `x86_64` in this context (ref: protobuf: https://github.com/protocolbuffers/protobuf/releases).

Reply:
Thanks, follow conventions from protobuf repo and zipped all the binaries as well; updated the bins attached to the 0.9.0 release